lightning-background-processor/src/lib.rs

```diff
@@ -609,7 +609,7 @@ mod tests {
 
 	const EVENT_DEADLINE: u64 = 5 * FRESHNESS_TIMER;
 
-	#[derive(Clone, Eq, Hash, PartialEq)]
+	#[derive(Clone, Eq, Hash, PartialEq, Eq)]
```

Comment:
This already derived `Eq`, no change needed here.

Reply:
FIxed: https://github.com/lightningdevkit/rust-lightning/pull/1763/commits/eed12878cb5e6160e8d9bbc7079e47d9823e0d5a